db/schema.rb

```diff
@@ -333,6 +333,7 @@
     t.citext "support_identifier", null: false
     t.boolean "is_test"
     t.integer "school_type", default: 0, null: false
+    t.boolean "using_openstax", default: false
```

Comment:
I was going to suggest that you rename this column to `is_usinng_openstax` because that is the Rails convention but... I see that `using_openstax` is already being used all throughout the codebase (I don't know why) so... I'll leave it up to you to decide.

Reply:
Leaving it as is.